identity/backend/Makefile

```diff
@@ -0,0 +1,32 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+build:
+	mvn clean package
+
+install:
+	mvn clean install
+
+license-check:
+	mvn license:check
+
+license-fix:
+	mvn license:format
+
+test: license-check
+	mvn clean verify
+
+generate-dependency-docs:
```

Comment:
❌ I'm not sure that we will have dedicated dependency list for Identity and would expect it being part of the monojar build. So I would remove this script as well as the template